lib/core/device.js

```diff
@@ -217,7 +217,11 @@ device.detectUsb = function(driverId, cb) {
     vendorId: driverManifest.vendorId,
     productId: driverManifest.productId
   };
-  chrome.usb.getDevices(identification, function(results) {
```

Comment:
I think this is something that we need to look into for Chrome 46, as it looks like Google does want to improve `chrome.serial.getDevices`: https://code.google.com/p/chromium/issues/detail?id=522217

If they implement that, we can get the vendor ID and product ID using `chrome.serial.getDevices`, which will be the right way of doing things.